### PR TITLE
Untested changes to allow use of a proxy server

### PR DESCRIPTION
**Pull request by peeto**
Requires testing before merging

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "willvincent/feeds",
+    "name": "peeto/feeds",
```
Comment (willvincent) [inline composer.json]: This is preventing a merge of this PR.

**Review by willvincent — CHANGES_REQUESTED** [review]
I think the change to the configure method is still somewhat limiting. Rather than specfically defining a few additional options for proxy, instead it would be more flexible to expose an array that allows for defining _any_ curl option value(s).

Otherwise the next time someone comes up with another _thing_ to override there yet another round of changes and another release become necessary.

This current approach will ultimately make that configure() code is going to get much more complicated than it needs to be...

I think this can be improved, and it'll _never_ be merged with that composer.json change.

**Comment by peeto** [comment]
@willvincent,

The curloptions array had occurred to me. So I'll do that if we're in
agreement. The composer.json change was made purely so it could be tested
and will definitely be reverted.

**Comment by peeto** [comment]
The requested changes have been made. #40 has another request for a similar change to make() so based on the conversation here an $options parameter has been added. Please review changes but don't pull until #40 is resolved.

**Comment by peeto** [comment]
The latest comments in #40 suggest all this could have been avoided by using $feed->set_curl_options()

To mitigate wasting my time I've added the ability to change the strip tags and strip attributes in make().

So to sum up #41 contains the ability to set in the configuration CURL options (eg for proxy). It also contains a change to make() to allow the strip tags, strip attributes, and CURL options to be changed from their default (config) for an individual feed. The manner in which the changes to make() have been made allow for similar requests in the future to be made without further changes to the method signature.

**Comment by willvincent** [comment]
Looks good to me. Thanks :)

**Review by willvincent — APPROVED** [review]

**Pull request merged**